pkg/imgpkg/bundle/fake_images_metadata_builder_test.go

```diff
@@ -8,94 +8,53 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net/http/httptest"
+	"net/url"
 	"os"
 	"path/filepath"
 	"testing"
 
 	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/registry"
 	"github.com/google/go-containerregistry/pkg/v1"
 	regremote "github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/k14s/imgpkg/pkg/imgpkg/bundle"
 	"github.com/k14s/imgpkg/pkg/imgpkg/image"
-	"github.com/k14s/imgpkg/pkg/imgpkg/image/imagefakes"
+	"github.com/k14s/imgpkg/pkg/imgpkg/imagelayers"
 	"github.com/k14s/imgpkg/pkg/imgpkg/lockconfig"
+	"github.com/stretchr/testify/assert"
 )
 
-type FakeImagesMetadataBuilder struct {
-	state map[string]*ImageWithTarPath
-	t     *testing.T
+type FakeTestRegistryBuilder struct {
+	state  map[string]*ImageWithTarPath
```

Comment:
```suggestion
	images  map[string]*ImageWithTarPath
```

This is no longer the state but the list of images we are going to push to the registry.

Reply:
Good catch. Thx I made the change